openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -152,26 +172,33 @@ func (a *managementCPUsOverride) Admit(ctx context.Context, attr admission.Attri
 	}
 
 	if !a.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
-		return admission.NewForbidden(attr, fmt.Errorf("%s node and namespace caches not synchronized", PluginName))
+		return admission.NewForbidden(attr, fmt.Errorf("%s node or namespace or infra config cache not synchronized", PluginName))
 	}
 
-	nodes, err := a.nodeLister.List(labels.Everything())
+	clusterInfra, err := a.infraConfigLister.Get(infraClusterName)
```

Comment:
can one delete the infra resource? Would that brick the cluster?

Reply:
@dhellmann @browsell Does it expected that an admin can delete the infrastructure object? What additional components in a cluster relay on it?

Reply:
@sttts if it's deletable, it's a bug.  Admission shoudl be coded to prevent deletion of config.openshift.io objects.

Reply:
Nevermind, I misunderstood some things about the ordering during bootstrapping. It should be safe to assume the infrastructure resource exists when it's safe to create regular pods through the API.